model.js

```diff
@@ -17,8 +17,8 @@ const fetchAllArticles = () => {
   return database
     .query(
       `
-        SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) as comment_count
-        FROM articles
+        SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) as comment_count 
+        FROM articles 
         JOIN comments 
```

Comment:
I'd switch this to  LEFT JOIN so articles without comments don't get left out.

Reply:
good shout